### PR TITLE
moved readings to corresponding weeks

### PR DESCRIPTION
**Pull request by mkruijt**

### CODE REVIEW

File: Week0/README.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
-# Reading material for your first JavaScript lesson:
```
Comment (remarcmij) [inline Week0/README.md:1]: The link to this README from the main README is broken.

In addition the link to JavaScript Fundamentals README as the bottom of the main README is broken.

File: Week3/README.md [hunk 2]
```diff
@@ -1,52 +1,51 @@
...
-### Week 2
+Review the topics of week 2:
 
-- [Variables (var, let, const)](./../../../../fundamentals/blob/master/fundamentals/variables.md)
-- [Basic Data types (Strings, Numbers, Arrays, Booleans)](./../../../../fundamentals/blob/master/fundamentals/values.md)
+- [Variables](./../../../../fundamentals/blob/master/fundamentals/variables.md)
+- [Values](./../../../../fundamentals/blob/master/fundamentals/values.md)
 - [Operators](./../../../../fundamentals/blob/master/fundamentals/operators.md)
-- [Naming conventions](./../../../../fundamentals/blob/master/fundamentals/naming_conventions.md)
+- [Naming conventions](../fundamentals/naming_conventions.md)
```
Comment (remarcmij) [inline Week3/README.md]: You need some extra ../ to fix this broken link.

File: Week3/MAKEME.md [hunk 3]
```diff
@@ -193,7 +193,7 @@ Please make sure you REALLY understand the exercises below:
 
 _Deadline Sunday morning_
 
-Go through the reading material in the [README.md](/Week2/README.md) to prepare for your next class
+Go through the reading material in the [README.md](./../../../../JavaScript3/tree/master/Week1) to prepare for your next class
```
Comment (remarcmij) [inline Week3/MAKEME.md:196]: This link should point to the JavaScript2 repo README

**Review by remarcmij — CHANGES_REQUESTED** [review]
Just some links that need to be fixed.

File: Week0/README.md (hunk 1 above)
Comment (mkruijt) [reply Week0/README.md:1]: yes, that is correct, I removed the file and forgot to remove Week0 in the planning

**Review by remarcmij — APPROVED** [review]
Okidoki!

**Pull request merged**